### ISSUE TITLE
Simple full-text search on pages

### ISSUE THREAD

**Issue by lecunm** [KelkooGroup/theGardener#654]
Create one endpoint with basic search in parallel of the cache

Use Full text search  on documents

  Restict to :
     - only stable branch
     - first hierarchy found
     - do not get data included (OpenAPI, Scenario)

  Indexed document :
    - (080) hierarchy
    - ( X ) path
    - (030) branch
    - (100) label
    - (050) description
    - (030) pageContent (before processing of any kind)

  ( ) boost of the search

  Nice to have: Pagination

see [BDD Scenario](https://github.com/KelkooGroup/theGardener/blob/feature/654-simple-full-text-search-on-pages/test/features/search/search_through_name_pages.feature)
